assets/js/hooks/useInView.js

```diff
@@ -52,13 +52,22 @@ export const useInView = ( { sticky = false } = {} ) => {
 	const resetCount = useSelect( ( select ) =>
 		select( CORE_UI ).getInViewResetHook()
 	);
+	const forceInView = useSelect( ( select ) =>
+		select( CORE_UI ).getValue( 'forceInView' )
+	);
```

Comment:
This would force all uses of the hook to be activated all at once, why would we want/need to do that?

Reply:
The 2nd and 3rd steps of the feature tour that point to a widget that may not be loaded yet because the top pages widget is not in view yet. That's why we need a way to force all sections to come in view on demand.

Reply:
Sure, it's understandable that we'd need to force something to be inView for the tour; I'm specifically questioning whether we should be forcing everything on the whole dashboard to be in view rather than only the relevant area?

Reply:
It's way more complicated if we want to make in view only specific sections and may cause layout shifts when we transition from one step to another. So, I think it's better to make the whole dashboard to be in view for this one time edge case.